packages/template-retail-react-app/app/components/item-variant/item-attributes.test.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import ItemVariantProvider from '@salesforce/retail-react-app/app/components/item-variant'
+import {renderWithProviders} from '@salesforce/retail-react-app/app/utils/test-utils'
+import {screen, waitFor} from '@testing-library/react'
+import ItemAttributes from '@salesforce/retail-react-app/app/components/item-variant/item-attributes'
+import {getDisplayVariationValues} from '@salesforce/retail-react-app/app/utils/product-utils'
+import PropTypes from 'prop-types'
+import {rest} from 'msw'
+import {
+    mockBundledProductItemsVariant,
+    mockProductBundleWithVariants,
+    mockProductBundle
+} from '@salesforce/retail-react-app/app/mocks/product-bundle'
```

Comment:
What's the difference between these two? I scanned the data and had trouble differentiating them (because WHOA lots of data)

`mockBundledProductItemsVariant` vs `mockProductBundleWithVariants`

I want to make sure there's not opportunity to go from this:

```
const myFirstMock = {
  id: 'blah',
  someKey: 'someVal',
  etc: 'etc',
  qty: 1,
}

const myVerySimilarMock = {
  id: 'blah',
  someKey: 'someVal',
  etc: 'etc',
  qty: 2,
}
```

to this:
```
const myMockFactory = ({ qty }) => {
  id: 'blah',
  someKey: 'someVal',
  etc: 'etc',
  qty
}
```